### DISCUSSION TITLE
section '[FluidProperty]' does not have an associated "Action".

### DISCUSSION THREAD

**Discussion by SunJianhang678** [idaholab/moose#28913] [Q&A Modules: General] [answered]
hello,everyone! I'm trying to develop an application, in which the Water97FluidProperties is used. But after the "FluidProperties" block is included in the input file, it runs into an error like "section '[FluidProperty]' does not have an associated "Action"." How can I solve this problem? Thanks very much!
![image](https://github.com/user-attachments/assets/c330ce19-5db7-4bfc-be81-3cb9841e951b)
[1dpipeT97.txt](https://github.com/user-attachments/files/17489507/1dpipeT97.txt)

**Comment by GiudGiud** [comment]
Hello

It should be `[FluidProperties]` not FluidProperty

**Reply by SunJianhang678** [reply]
![image](https://github.com/user-attachments/assets/58110582-d128-4ed9-9471-89b893c361e7)
Thanks! I've tried that and it didn't work. I've also tried to set [FluidProperties] as a sub-block of [Modules], which didn't work either. I'm wondering if I need some settings like a specific "action" or turn on a switch in the Makefile to use the [FluidProperties] in an inputfile? Thanks for your patience!

**Reply by GiudGiud** [reply, marked as answer]
You also need to turn in the fluid properties module in the Makefile , then make clobberall and recompile

**Reply by SunJianhang678** [reply]
I cleared the build directory and recompiled, then it didn't give this error reporting again. Thank you very much!
